pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -70,29 +66,21 @@ func (o *objectWatcherImpl) Create(clusterName string, desireObj *unstructured.U
 		return err
 	}
 
-	fedKey, err := keys.FederatedKeyFunc(clusterName, desireObj)
+	clusterObj, err := dynamicClusterClient.DynamicClientSet.Resource(gvr).Namespace(desireObj.GetNamespace()).Create(context.TODO(), desireObj, metav1.CreateOptions{})
 	if err != nil {
-		klog.Errorf("Failed to get FederatedKey %s, error: %v", desireObj.GetName(), err)
-		return err
-	}
-	_, err = helper.GetObjectFromCache(o.RESTMapper, o.InformerManager, fedKey)
-	if err != nil {
-		if !apierrors.IsNotFound(err) {
-			klog.Errorf("Failed to get resource %v from member cluster, err is %v ", desireObj.GetName(), err)
-			return err
-		}
-
-		clusterObj, err := dynamicClusterClient.DynamicClientSet.Resource(gvr).Namespace(desireObj.GetNamespace()).Create(context.TODO(), desireObj, metav1.CreateOptions{})
-		if err != nil {
-			klog.Errorf("Failed to create resource(kind=%s, %s/%s) in cluster %s, err is %v ", desireObj.GetKind(), desireObj.GetNamespace(), desireObj.GetName(), clusterName, err)
-			return err
+		if apierrors.IsAlreadyExists(err) {
+			klog.Infof("Skip creating resource(kind=%s, %s/%s) in cluster %s as resource is already existing", desireObj.GetKind(), desireObj.GetNamespace(), desireObj.GetName(), clusterName)
```

Comment:
`Skip creating resource(kind=%s, %s/%s) in cluster %s because it already exists`

Reply:
Done.